src/mongo/delegates/Updates.class.php

```diff
@@ -865,7 +901,7 @@ protected function lockAllDocuments($subjectsOfChange, $transaction_id, $context
             }else{
 
                 if(count($lockedSubjects)) //If any subject was locked, unlock it
-                $this->unlockAllDocuments($transaction_id);
+                    $this->unlockAllDocuments($transaction_id);
```

Comment:
Can we wrap this conditional in curly braces?  These kind of ifs make me a little uncomfortable if they're not explicitly one-liners.